shim/opentelemetry-opentracing-shim/src/opentelemetry/shim/opentracing_shim/util.py

```diff
@@ -52,3 +63,19 @@ def event_name_from_kv(key_values):
         return DEFAULT_EVENT_NAME
 
     return key_values["event"]
+
+
+def opentracing_to_opentelemetry_kind_tag(opentracing_tags: Attributes):
+    """A helper function to extract opentelmetry Kind from opentracing tag"""
+    kinds = {
```

Comment:
Can this be defined outside of this function?

Reply:
I don't see the exact need as this will not be used for the conversion but if you have suggestion about where to put I am ok to move it

Reply:
Well we don't really want to keep redefining the mapping every time this function is called. Could be defined as a private constant like `_KIND_MAPPING = { ... }` at the top of the file under `DEFAULT_EVENT_NAME `.